dvc/repo/experiments/__init__.py

```diff
@@ -450,7 +448,7 @@ def new(
         """
         if checkpoint_resume is not None:
             return self._resume_checkpoint(
-                *args, **kwargs, checkpoint_resume=checkpoint_resume
+                *args, checkpoint_resume=checkpoint_resume, **kwargs
```

Comment:
Mypy was complaining that `checkpoint_resume` might be set twice. Not sure how, but changed the position and it stopped.